.builds/openbsd.yml

```diff
@@ -31,3 +31,8 @@ tasks:
     cmake -G Ninja ..
     cmake --build . --config Debug
     ./bin/nvim --version
+- test: |
+    export LC_CTYPE=en_US.UTF-8
+    cd neovim
+    # oldtests
+    gmake -C src/nvim/testdir NVIM_PRG=`pwd`/build/bin/nvim
```

Comment:
referencing the Windows (AppVeyor) [build.ps1](https://github.com/neovim/neovim/blob/master/ci/build.ps1#L121), this could be:

    cmake --build . --config Debug --target oldtest

Reply:
Agreed, but I'm enabling the tests in stages so that we can tackle this incrementally.